source/agora/node/Ledger.d

```diff
@@ -66,26 +66,28 @@ public class Ledger
     {
         if (!tx.verify(&this.findOutput))
             return false;
+
         this.storage ~= tx;
-        if (this.storage.length >= 8)
+        if (this.storage.length == Block.TxsInBlock)
```

Comment:
Looks like the commit is doing much more than "Encode the block size in the Block and use it"